src/main/java/teammates/ui/webapi/SendLoginEmailAction.java

```diff
@@ -0,0 +1,60 @@
+package teammates.ui.webapi;
+
+import static teammates.common.util.FieldValidator.REGEX_EMAIL;
+
+import teammates.common.util.Const;
+import teammates.common.util.EmailSendingStatus;
+import teammates.common.util.EmailWrapper;
+import teammates.common.util.StringHelper;
+import teammates.ui.output.SendLoginEmailResponseData;
+import teammates.ui.request.InvalidHttpRequestBodyException;
+
+/**
+ * Sends a login email.
+ */
+class SendLoginEmailAction extends Action {
+
+    @Override
+    AuthType getMinAuthLevel() {
+        return AuthType.PUBLIC;
+    }
+
+    @Override
+    void checkSpecificAccessControl() throws UnauthorizedAccessException {
+        // Sending of login emails can be requested by anyone
+    }
+
+    @Override
+    public JsonResult execute() throws InvalidHttpRequestBodyException, InvalidOperationException {
+        String userEmail = getNonNullRequestParamValue(Const.ParamsNames.USER_EMAIL);
+        if (!StringHelper.isMatching(userEmail, REGEX_EMAIL)) {
+            throw new InvalidHttpParameterException("Invalid email address: " + userEmail);
+        }
+
+        String userCaptchaResponse = getRequestParamValue(Const.ParamsNames.USER_CAPTCHA_RESPONSE);
+        if (!recaptchaVerifier.isVerificationSuccessful(userCaptchaResponse)) {
+            return new JsonResult(new SendLoginEmailResponseData(false, "Something went wrong with "
+                    + "the reCAPTCHA verification. Please try again."));
```

Comment:
"Something went wrong" often implies that the fault lies within the system, whereas this response is sent for all reasons of failure (including null or wrong CAPTCHA). If we cannot differentiate the reason, better to say "ReCAPTCHA verification failed. Please try again."; although the meaning is the same, the tone is different.